bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml

```diff
@@ -66,3 +66,21 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: NBDBLongElectionTimer
+      annotations:
+        message: |
+          OVN NB DB has a high election timer interval.
```

Comment:
nitpick: "NB DB" -> "NB DB raft cluster"

Reply:
will fix that :-)